unity-client/Assets/Builder/Scripts/Gizmos/DCLBuilderGizmoManager.cs

```diff
@@ -153,7 +161,7 @@ private void Update()
             }
         }
 
-        private void SetGizmoType(string gizmoType)
+        public void SetGizmoType(string gizmoType)
```

Comment:
Since right now there are different components that are using gizmos type, I think that the responsibility to set the gizmos type should be from those components. If we use the event approach, the manager will start to listen to a lot of events that will make the maintainability of the code more difficult